recipes-hulks/hula/hula.bb

```diff
@@ -0,0 +1,226 @@
+SUMMARY = "hula"
+HOMEPAGE = "https://hulks.de"
+LICENSE = "GPL-3.0-only"
+LIC_FILES_CHKSUM = "file://${WORKDIR}/git/COPYING;md5=5b4473596678d62d9d83096273422c8c"
+
+inherit cargo
+
+SRC_URI += "git://github.com/julianschuler/hulk.git;branch=hula-dbus;protocol=https;"
+SRCREV = "d5dbc8746e9a4d4af5a1412f389f855d93ca575f"
+S = "${WORKDIR}/git/tools/hula"
+CARGO_SRC_DIR = "proxy"
+
+inherit pkgconfig
+
+DEPENDS += " \
+             dbus \
+             systemd \
+           "
+RDEPENDS:${PN} += " \
+                    dbus \
+                    systemd \
+                  "
+
+SYSTEMD_PACKAGES = "${PN}"
+SYSTEMD_SERVICE:${PN} = "hula.service"
+SRC_URI += " \
+             file://hula.service \
+             file://org.hulks.hula.conf \
+           "
+
+inherit systemd
+
+do_install:append() {
+  install -d ${D}${systemd_unitdir}/system/
+  install -m 0644 ${WORKDIR}/hula.service ${D}${systemd_unitdir}/system/
+  install -d ${D}/usr/share/dbus-1/system.d/
+  install -m 0644 ${WORKDIR}/org.hulks.hula.conf ${D}/usr/share/dbus-1/system.d/
```

Comment:
There might be a variable in bitbake to point to `/usr/share/`... I think it might be `$datadir` IIRC

Reply:
Thanks, fixed.